components/automate-cli/pkg/verifyserver/constants/checkconstant_test.go

```diff
@@ -9,22 +9,42 @@ import (
 func TestGetBastionChecks(t *testing.T) {
 	resp := GetBastionChecks()
 	assert.Equal(t, len(resp), 3)
-	assert.Equal(t, resp[0], HARDWARE_RESOURCE_COUNT)
-	assert.Equal(t, resp[1], CERTIFICATE)
-	assert.Equal(t, resp[2], SSH_USER)
+	assert.Contains(t, resp, HARDWARE_RESOURCE_COUNT)
+	assert.Contains(t, resp, CERTIFICATE)
+	assert.Contains(t, resp, SSH_USER)
+	assert.NotContains(t, resp, FIREWALL)
 }
 
 func TestGetRemoteChecks(t *testing.T) {
 	resp := GetRemoteChecks()
 	assert.Equal(t, len(resp), 10)
-	assert.Equal(t, resp[0], SYSTEM_RESOURCES)
-	assert.Equal(t, resp[1], SOFTWARE_VERSIONS)
-	assert.Equal(t, resp[2], SYSTEM_USER)
-	assert.Equal(t, resp[3], S3_BACKUP_CONFIG)
-	assert.Equal(t, resp[4], FQDN)
-	assert.Equal(t, resp[5], FIREWALL)
-	assert.Equal(t, resp[6], EXTERNAL_OPENSEARCH)
-	assert.Equal(t, resp[7], AWS_OPENSEARCH_S3_BUCKET_ACCESS)
-	assert.Equal(t, resp[8], EXTERNAL_POSTGRESQL)
-	assert.Equal(t, resp[9], NFS_BACKUP_CONFIG)
+	assert.Contains(t, resp, SYSTEM_RESOURCES)
+	assert.Contains(t, resp, SOFTWARE_VERSIONS)
+	assert.Contains(t, resp, SYSTEM_USER)
+	assert.Contains(t, resp, S3_BACKUP_CONFIG)
+	assert.Contains(t, resp, FQDN)
+	assert.Contains(t, resp, FIREWALL)
+	assert.Contains(t, resp, EXTERNAL_OPENSEARCH)
+	assert.Contains(t, resp, AWS_OPENSEARCH_S3_BUCKET_ACCESS)
+	assert.Contains(t, resp, EXTERNAL_POSTGRESQL)
+	assert.Contains(t, resp, NFS_BACKUP_CONFIG)
+}
+
+func TestGetCheckByType(t *testing.T) {
+	t.Run("Get NFS_BACKUP_CONFIG_MSG ", func(t *testing.T) {
+		resp := GetCheckByType(NFS_BACKUP_CONFIG)
+		assert.Equal(t, resp, NFS_BACKUP_CONFIG_MSG)
+	})
+
+	t.Run("Get FIREWALL_MSG", func(t *testing.T) {
+		resp := GetCheckByType(FIREWALL)
+		assert.Equal(t, resp, FIREWALL_MSG)
+	})
+
+	t.Run("Get FIREWALL_MSG", func(t *testing.T) {
```

Comment:
Rename the name for this, this is not a Firewall mesage check. And add test cases for other checks as well